### PR TITLE
feat(PP) : add search box for bundle id (#28508)

### LINKED ISSUE #28508: Add Search Box for Bundle ID
### Parent Issue

_No response_

### Task

Add a search box (similar to the one in the content search) that allows searching for a partial or complete bundle ID.

### Proposed Objective

Customer Support

### Proposed Priority

Same as Parent Issue

### Acceptance Criteria

- Bundle ID to search for could be on another page
- Partial Bundle IDs should be allowed
- Placeholder for the search box: `Bundle ID to search for...`

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by dsolistorres**
### Proposed Changes
* Added a search box for bundle ids

### Checklist
- [x] Tests

This PR fixes: #28508

### CODE REVIEW

**Review by erickgonzalez — APPROVED** [review]

**Review by jgambarios — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditAPIImpl.java [hunk 1]
```diff
@@ -290,30 +292,46 @@ public List<PublishAuditStatus> getAllPublishAuditStatus(Integer limit, Integer
...
+	 * limiting the number of assets of each one.
+	 * @param limit limit of rows for retrieved page
+	 * @param offset offset of rows for retrieved page
+	 * @param limitAssets max limit of assets to retrieve for each {@link PublishAuditStatus}
+	 * @param filter filter to apply to the query or null if no filter
+	 * @return List of {@link PublishAuditStatus}
+	 * @throws DotPublisherException if any error occurs
+	 */
+	@Override
+	public List<PublishAuditStatus> getPublishAuditStatus(
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditAPIImpl.java:309]: DbClosedIfOpened

File: dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditAPIImpl.java [hunk 2]
```diff
@@ -423,16 +441,48 @@ public Date getLastPublishAuditStatusDate() throws DotPublisherException {
...
+		return countPublishAuditStatus(null);
+	}
+
+	/**
+	 * Count filtered {@link PublishAuditStatus}
+	 * @param filter filter to apply to the query or null if no filter
+	 * @return number of rows that match the filter
+	 * @throws DotPublisherException if any error occurs
+	 */
+	@Override
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditAPIImpl.java:453]: DBClosedIfOpened

File: dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditAPIImpl.java (hunk 1 above)
Comment (erickgonzalez) [reply dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditAPIImpl.java:310]: Done here https://github.com/dotCMS/core/pull/29149/commits/69cdc74fc7c41969d3d05a530e642708a8000473

File: dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditAPIImpl.java (hunk 2 above)
Comment (erickgonzalez) [reply dotCMS/src/main/java/com/dotcms/publisher/business/PublishAuditAPIImpl.java:455]: Done here https://github.com/dotCMS/core/pull/29149/commits/69cdc74fc7c41969d3d05a530e642708a8000473

**Review by jdotcms — APPROVED** [review]

**Pull request merged**